Core/APIRequest.swift

```diff
@@ -34,43 +33,73 @@ public class APIRequest {
         
     }
     
+    public enum HTTPMethod: String {
+        case get = "GET"
+        case head = "HEAD"
+        case post = "POST"
+        case put = "PUT"
+        case delete = "DELETE"
+        case connect = "CONNECT"
+        case options = "OPTIONS"
+        case trace = "TRACE"
+        case patch = "PATCH"
+    }
+    
     @discardableResult
     public static func request(url: URL,
                                method: HTTPMethod = .get,
-                               parameters: [String: Any]? = nil,
-                               completion: @escaping APIRequestCompletion) -> Request {
+                               parameters: [String: String]? = nil,
```

Comment:
I change this to only accept strings since we were only ever using strings, and whilst Alamofire accepted other things, it wouldn't successfully add any parameter that isn't a string to the URL.